scripts/install_environment.sh

```diff
@@ -45,14 +45,11 @@ function config_additional_modules() {
 
 function install_libvirt() {
     echo "Installing libvirt-related packages..."
-    source "/etc/os-release" # This should set `PRETTY_NAME` as environment variable
-    if [[ "${PRETTY_NAME}" == "Rocky Linux 9"* ]]; then
-        sudo dnf install -y 'dnf-command(config-manager)'
-        sudo dnf config-manager --set-enabled crb
-    fi
 
-    # CRB repo is required for libvirt-devel
-    sudo dnf install -y --enablerepo=crb \
+    # CRB repo is required for libvirt-devel in some versions of RHEL
+    sudo dnf install -y 'dnf-command(config-manager)' || true
+    sudo dnf config-manager --set-enabled crb || true
+    sudo dnf install -y \
```

Comment:
just a recomandation when using dnf you can add the following
```console
sudo dnf install -y \
     --setopt=tsflags=nodocs --setopt=install_weak_deps=False \
     ...
 ```
 
 This will remove a lot of unneeded packages and docs from the installation - make it faster and smaller

Reply:
Thank you